patches/minecraft/net/minecraft/client/settings/GameSettings.java.patch

```diff
@@ -24,19 +24,18 @@
          try
          {
              if (!this.field_74354_ai.exists())
-@@ -887,6 +890,11 @@
+@@ -886,6 +889,10 @@
+                     {
                          if (s1.equals("key_" + keybinding.func_151464_g()))
                          {
-                             keybinding.func_151462_b(Integer.parseInt(s2));
-+                            if (s2.indexOf(':') != -1)
-+                            {
```

Comment:
Actually yes, it should of, see the rest of the codebase braces are on new lines.